jbmc/src/java_bytecode/java_object_factory_parameters.cpp

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************\
+
+Module:
+
+Author: Daniel Poetzl
+
+\*******************************************************************/
+
+#include "java_object_factory_parameters.h"
```

Comment:
Why is this in a commit called "move object factory parameters to util"?

Reply:
Ok, renamed the commit to "Move object factory parameters to util and add java object factory parameters" (plus a body with some more details).